Formula/nrpe.rb

```diff
@@ -45,6 +45,18 @@ def install
     system "make", "install-daemon-config"
   end
 
+  plist_options :manual => "nrpe -n -c #{HOMEBREW_PREFIX}/etc/nrpe.cfg -d"
+
+  def caveats
+    <<-EOS.undent
+    The nagios plugin check_nrpe has been installed in:
+      #{HOMEBREW_PREFIX}/sbin
+
+    You can start the daemon with
+      #{bin}/nrpe -c #{etc}/nrpe.cfg -d
```

Comment:
If this is the same as the `:manual` invocation given to `plist_options`, it can be removed; we print the `:manual` command in the caveats.